lib/ex_fizz_buzz_enterprise/input_worker.ex

```diff
@@ -14,7 +14,6 @@ defmodule ExFizzBuzzEnterprise.InputWorker do
   
   def init(args) do
```

Comment:
General style - initialize the state inside the `init` function, keep the `start_link` -> `init` pipe for setup arguments.